presto-clp/src/main/java/com/yscope/presto/ClpColumnHandle.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yscope.presto;
```

Comment:
Based on the package name for other plugins/connectors, I think the package should really be `com.facebook.presto.plugin.clp`, right? Or is there a doc somewhere saying that we should use the current form of the package name?

Reply:
Yeah, that needs to be fixed.